models/schema.big-iot.org-mobility.ttl

```diff
@@ -0,0 +1,2392 @@
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix core: <http://schema.big-iot.org/core/> .
+@prefix common: <http://schema.big-iot.org/common/> .
+@prefix mobility: <http://schema.big-iot.org/mobility/> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix environment: <http://schema.big-iot.org/environment/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+@prefix ns: <http://creativecommons.org/ns#>.
+@prefix ssn: <http://purl.oclc.org/NET/ssnx/ssn#>.
+@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#>.
+@prefix qudt: <http://data.qudt.org/qudt/owl/1.0.0/unit.owl#>.
+@prefix dul: <http://www.loa.istc.cnr.it/ontologies/DUL.owl#>.
+@prefix time: <http://www.w3.org/2006/time#>.
+@prefix vann: <http://purl.org/vocab/vann/>.
+
+
+
+mobility: dc:contributor "Achille Zappa, NUIG, Galway"@en,
+                           "Hoan Quoc, NUIG, Galway"@en,
+                           "Aparna Saisree, Siemens AG, Munich"@en,
+                           "Darko Anicic, Siemens AG, Munich"@en,
+                           "Yulia Svetashova, Bosch CR, Stutgard"@en;
+                           
+            dc:description " Mobility Model for Mobility Domain terminology annotations. We further acknowledge, Pilot providers and other open call projects for their valuable comments. Please report any errors to achille.zappa@insight-centre.org. "@en;
+            dc:rights "Copyright EU H2020 BIG IoT";
```

Comment:
Hi Achille,

as far as I know it is not possible to claim the Copyright for "EU H2020 BIG IoT" because it is not a legal entity.
It should be a legal entity like NUIG, Siemens, Bosch (maybe all together).